internal/sidecarlogresults/sidecarlogresults.go

```diff
@@ -183,10 +240,19 @@ func parseResults(resultBytes []byte, maxResultLimit int) (result.RunResult, err
 	if len(resultBytes) > maxResultLimit {
 		return runResult, fmt.Errorf("invalid result \"%s\": %w of %d", res.Name, ErrSizeExceeded, maxResultLimit)
 	}
+	var resultType result.ResultType
+	switch res.Type {
+	case taskResultType:
+		resultType = result.TaskRunResultType
+	case stepResultType:
+		resultType = result.StepResultType
+	default:
+		return result.RunResult{}, fmt.Errorf("Invalid sidecar result type %v. Must be %v or %v", res.Type, taskResultType, stepResultType)
```

Comment:
Is this error message user facing (error should not start with uppercase if not)?

Reply:
Agreed. It is not. Thanks. Will fix.